src/ChunkMap.h

```diff
@@ -492,27 +431,26 @@ class cChunkMap
 		}
 	};
 
-	typedef std::list<cChunkLayer *> cChunkLayerList;
-
-	typedef std::list<cChunkStay *> cChunkStays;
-
-	/** Finds the cChunkLayer object responsible for the specified chunk; returns nullptr if not found. Assumes m_CSLayers is locked. */
-	cChunkLayer * FindLayerForChunk(int a_ChunkX, int a_ChunkZ);
+	struct ChunkCoordinate
+	{
+		bool operator() (const ChunkCoordinate & a_Lhs, const ChunkCoordinate & a_Rhs) const
```

Comment:
I was just hoping to reuse the struct for both storage and comparison, so I wouldn't need to create an extra ChunkCoordinateComparer.